content/docs/setup/kubernetes/istio-cni-install/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Istio CNI Installation
+description: Instructions for installing and using Istio with the Istio CNI plugin to allow Istio users to deploy services via lower privileged RBAC.  The Istio CNI plugin removes the requirement for the privileged, `NET_ADMIN` initialization container in the users' pods in the mesh.
+weight: 70
+keywords: [kubernetes,cni,sidecar,proxy,network,helm]
+---
+
+By default Istio injects an `initContainer`, `istio-init`, in pods deployed in the mesh.  The `istio-init`
+container sets up the pod network traffic redirection to/from the Istio sidecar proxy.  This requires the user
+or service-account deploying pods to the mesh to have high enough Kubernetes RBAC permissions to deploy
+[`NET_ADMIN` containers](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container).  Requiring Istio users to have such high Kubernetes RBAC permissions is problematic for
+some organizations' security compliance.  The Istio CNI plugin is a replacement for the `istio-init`
```

Comment:
Add a reference to CNI?

The Istio [CNI (Container Network Interface)](https://github.com/containernetworking/cni/blob/master/README.md) plugin is a replacement ...

Reply:
IMO, the main CNI specification pointer isn't as valuable here and might be misleading since the thing we're talking about is the "Istio CNI plugin".  There's a reference to the Istio CNI repo here and the containernetworking content in the prerequisites section to clarify the general CNI related settings.

Reply:
I agree that a link to the Istio CNI plugin is good, but what's still missing here (and seems also missing in the "Istio CNI plugin" reference) is an indication of what CNI is. The first use of the acronym should be expanded out:

CNI (Container Network Interface)

and a reference to something [like this](https://github.com/containernetworking/cni/blob/master/README.md#cni---the-container-network-interface), either here or in the Istio CNI plugin doc would be good.

Reply:
Cool... adding it into this installation doc.